Reorganize  the command interfaces

JuneZhou19 opened the pull request:
Reorganize  the command interfaces to avoid misleading.
@fub2 @xiar

JuneZhou19 committed:
Reorganize  the command interfaces

 Reorganize  the command interfaces to avoid misleading.

fub2 reviewed (changes requested):
a few changes reqired.

JuneZhou19 committed:
Update get_start.rst

To avoid user to mis-understand of running the commands in sequence, make the command interfaces into bullets list.
